command/helper/config.go

```diff
@@ -221,3 +238,22 @@ func readConfigFile(path string) (*Config, error) {
 
 	return &config, nil
 }
+
+func parseDNSString(s string) (version string, name string) {
+	s = strings.Trim(s, "/")
+	fmt.Println(s)
+	split := strings.Split(s, "/")
+	if len(split) <= 1 {
+		return
+	}
+	switch split[0] {
+	case "dns":
+		version = "dns"
+	case "dns4":
+		version = "dns4"
+	case "dns6":
+		version = "dns6"
+	}
+	name = split[1]
+	return
+}
```

Comment:
I would appreciate it if this func had error handling and tests written for it.
In my experience, we should never expect a string to be the correct format, as it's not always the case.

Reply:
@zivkovicmilos Here is the commit [link](https://github.com/0xPolygon/polygon-sdk/pull/191/commits/bb373293bbfc4c97524dbc250daa0dae653e3d10)

Reply:
@munna0908 we're missing the tests that @zivkovicmilos  mentioned. We don't have any test cases for bogus DNS names.

Reply:
@munna0908 
I think it would be wise if we added a regex check to the func as well, this way we can keep the format air-tight

Reply:
@zivkovicmilos @lazartravica  Added regex to match the dns string commit [link](https://github.com/0xPolygon/polygon-sdk/pull/191/commits/fd76aaf24fdfdb8156ee2f600f92f6826f339e07)

Reply:
Added test cases as well commit [link](https://github.com/0xPolygon/polygon-sdk/pull/191/commits/f4307b4977f08088f65e675dc29a5c27938b0931)